SPECS/intel-pf-bb-config/intel-pf-bb-config.spec

```diff
@@ -0,0 +1,53 @@
+%global inih_version 44
+Summary:        A tool for configuring certain Intel baseband devices
+Name:           intel-pf-bb-config
+Version:        21.11
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Tools
+URL:            https://github.com/intel/pf-bb-config
+Source0:        https://github.com/benhoyt/inih/archive/r%{inih_version}.tar.gz#/inih-%{inih_version}.tar.gz
```

Comment:
We should separate out `inih` to a separate package. The way it's being pulled in here makes it **very** hard to track the package for CVEs.

There's also the choice of upgrading `intel-pf-bb-config` to 22.03, [which gets rid of the `inih` dependency](https://github.com/intel/pf-bb-config/blob/9ed477ad7c15a128154a32778083555112cf2558/rel_notes.txt#L4-L9), but I suspect there's a good reason you're pulling in this specific version.

Reply:
Updated spec to 22.03